packages/mobile/src/language/Language.test.tsx

```diff
@@ -22,7 +22,9 @@ describe('Language', () => {
     })
 
     fireEvent.press(getByText('Español (América Latina)'))
-    expect(mockNavigation.navigate).toHaveBeenCalledWith(Screens.JoinCelo)
+    // Run timers, because Touchable adds some delay
```

Comment:
I guess, we can remove this now, cause https://github.com/celo-org/celo-monorepo/pull/4102 is merged 🙂

Reply:
Yep good catch